src/main/java/org/sagebionetworks/EvaluationUtils.java

```diff
@@ -412,7 +562,12 @@ private static String formatInterval(final long l) {
 	}
 
 	public static Long getTimeRemaining(SubmissionStatus ss) {
-		return getLongAnnotation(ss, TIME_REMAINING);
+		Long longV2 = getLongAnnotationV2(ss, TIME_REMAINING);
```

Comment:
I think it is too early for this kind of change, we first need to make sure to apply the V2 annotations and backfill if needed in the future, later we can switch completely to the new annotations also in the orchestrator.